sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -6353,7 +6364,7 @@ namespace Azure { namespace Storage { namespace Blobs {
         struct AcquireBlobLeaseOptions
         {
           Azure::Core::Nullable<int32_t> Timeout;
-          int32_t LeaseDuration = -1;
+          std::chrono::seconds LeaseDuration;
```

Comment:
FYI, it looks like you have changed the default, from infinite to immediate.

Reply:
The default value of this variable will be overwritten anyway https://github.com/Azure/azure-sdk-for-cpp/blob/d5a53423bd125f2306f00da3f2ee94ce03be8043/sdk/storage/azure-storage-blobs/src/blob_lease_client.cpp#L23.

The `= -1` was to suppress compiler warning.